dynamodb/dynamodb.ts

```diff
@@ -100,9 +100,9 @@ const processRecords = async (endpoint, tableName, handler, options) => {
     try {
       const { Streams } = await client.listStreams({ TableName: tableName });
       await Promise.all(
-        Streams.map(({ StreamArn }) =>
```

Comment:
we should:

```ts
import type extern from "./dynamodb.extern";
```

And use the `: extern["foo"]` type annotations to enforce typing.

Reply:
Had to add some ts-ignore's to the DynamoDB clients.